### ISSUE TITLE
Get all companies of a city (with zipCode)

### ISSUE THREAD

**Issue by QuentinSu** [etalab/sirene_as_api#217]
Hi!

First thanks for your work!
Can i get all the companies of a given city ? It's actually not possible, i think with [...]/full_text/[zipcode]
as code_postal is not on :text possibility.

Can i have a way to get this ?

Thanks again

**Comment by Samuelfaure** [comment]
Hello !

We are releasing new endpoints in production, which can be filtered by any parameter. Be sure to subscribe to the newsletter on the main page of [entreprise.data.gouv.fr](https://entreprise.data.gouv.fr)  to get informations about the latest releases.

The new documentation is getting approved before release, you can have a sneak peek here :+1: 
[https://github.com/etalab/entreprise.data.gouv.fr/blob/add_v3_doc/src/components/apiDoc/apiDocSirene/ApiDocSireneEndpointsV3.vue](https://github.com/etalab/entreprise.data.gouv.fr/blob/add_v3_doc/src/components/apiDoc/apiDocSirene/ApiDocSireneEndpointsV3.vue)

TL;DR : you want to call `/v3/unites_legales/?code_postal=75011` or `/v3/etablissements/?code_postal=75011` for a filtered index of those endpoints.

**This will return all the companies in 75011.** This is NOT a full-text search, which is not available for the new version yet.

If you need to do a full text search and then filter, `/v1/full_text/<MY_TEXT>?code_postal=75011` should work, but is considered deprecated.
